tests/moe/test_unified_moe_cutlass.py

```
@@ -336,6 +469,188 @@ def test_cutlass_runner_rejects_out_of_scope_configs(config, match):
+        runner.check_support()
+
+
+def test_cutlass_fp8_block_rejects_cuda_below_12_8(monkeypatch):
+    monkeypatch.setattr(
+        "flashinfer.jit.cpp_ext.is_cuda_version_at_least",
+        lambda _version: False,
+    )
+    runner = CutlassFp8BlockRunner.__new__(CutlassFp8BlockRunner)
+    runner.config = _config(quant=QuantConfig(variant=QuantVariant.DeepSeekFp8))
+    runner._device_arch = 90
+    with pytest.raises(NotImplementedError, match="CUDA 12.6 or lower"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)